frontend/src/pages/home.tsx

```diff
@@ -0,0 +1,81 @@
+import { Flex, GridItem } from "@chakra-ui/react";
+import Hero from "../components/Hero";
+import FeatureSection from "../components/FeatureSection";
+import SubFeatureSection from "../components/SubFeatureSection";
+import MetricSection from "../components/MetricSection";
+
+const Home = () => {
+  //Props stored here until we start implementing an appropriate store
+  const heroProps = {
+    headline: "We ship coders.",
+    content:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+    buttonLabel: "Lorem Ipsum",
+    buttonUrl: "/loremipsum",
+  };
+
+  const featureProps = {
+    headline: "Lorem Ipsum",
+    content:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.",
+    buttonLabel: "Lorem Ipsum",
+    buttonUrl: "/loremipsum",
+  };
+
+  const subFeaturesProps = [
+    {
+      headline: "Lorem Ipsum 1",
+      content:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+      linkLabel: "Lorem Ipsum 1 >",
+      url: "/loremipsum1",
+    },
+    {
+      headline: "Lorem Ipsum 2",
+      content:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+      linkLabel: "Lorem Ipsum 2 >",
+      url: "/loremipsum2",
+    },
+    {
+      headline: "Lorem Ipsum 3",
+      content:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+      linkLabel: "Lorem Ipsum 3 >",
+      url: "/loremipsum3",
+    },
+  ];
+
```

Comment:
Danny has moved these into their own `constants` file. Import them from there instead of re declaring them

Reply:
This was an error. For some reason my push had retained an old home.tsx that wasn't in it's own directory. I've removed it to match prod. Everything should be fine now